src/menu/host_interface.c

```diff
@@ -149,6 +150,10 @@ void main_menu_host_interface(engine_ctx_t *ctx,
       near_main(usb_evt);
       break;
     }
+    case 9: {
```

Comment:
Move app id for solana to 9 (Coin_Type + 1, as 1 is given to manager app)

Reply:
updated